docs/en/DevelopingUFSExtensions.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: global
 group: Resources
-title: Developing Under Storage Extensions
+title: Develop Under Storage Extensions
```

Comment:
to stay consistent with tense I think it should be Developing Under Storage Extensions. If we switch to imperative tense, we should switch for all the top-level doc titles.